src/content/components/ProfileCallTreeSettings.js

```diff
@@ -27,17 +27,21 @@ class ProfileCallTreeSettings extends Component {
   }
 
   render() {
-    const { jsOnly, invertCallstack, searchString } = this.props;
+    const { implementationFilter, invertCallstack, searchString } = this.props;
     return (
       <div className='profileCallTreeSettings'>
         <ul className='profileCallTreeSettingsList'>
           <li className='profileCallTreeSettingsListItem'>
             <label className='profileCallTreeSettingsLabel'>
-              <input type='checkbox'
+              Filter:
+              <select
```

Comment:
the select has a too big height on Linux

![capture d ecran de 2017-04-20 17-03-54](https://cloud.githubusercontent.com/assets/454175/25237604/6861d1ce-25eb-11e7-967c-d497ca15bdd8.png)